extractor/src/main/java/org/schabi/newpipe/extractor/Extractor.java

```diff
@@ -83,7 +83,6 @@ public String getId() throws ParsingException {
      * @return the name
      * @throws ParsingException if the name cannot be extracted
      */
-    @Nonnull
```

Comment:
When can the name be `null`?

Reply:
It can be null in most of extractors, because the `getString(String key)` method of nanojson is used in several places in the JSON objects extracted.

But the risk that the name can be null is not really big. Should I readd the annotation, remove it in the extractors on which the name can be null or return an empty string as the default value (so the values returned will be, in this case, not null), or should this be done in a separate PR (and maybe not only for `getName` method)?